CMakeLists.txt

```diff
@@ -1,83 +1,24 @@
 cmake_minimum_required(VERSION 3.16 FATAL_ERROR)
 
-IF(DEFINED ENV{MR_USE_CPP_23} AND "$ENV{MR_USE_CPP_23}" STREQUAL "ON")
-  set(MR_CXX_STANDARD 23)
-  message("enable c++23")
-ELSE()
-  set(MR_CXX_STANDARD 20)
-ENDIF()
-set(CMAKE_CXX_STANDARD ${MR_CXX_STANDARD})
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules")
+include(CompilerOptions)
```

Comment:
should `include(DetectPlatform)` be called before this line?

Reply:
`DetectPlatform` uses `CMAKE_SYSTEM_PLATFORM` variable which is unset before the `project(MeshLib CXX)` declaration.